pkgs/by-name/co/codespell/package.nix

```diff
@@ -1,10 +1,10 @@
 { lib
 , fetchFromGitHub
 , aspellDicts
-, python3
+, python3Packages
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python3Packages.buildPythonApplication rec {
```

Comment:
Is that required for this change? Archivebox itself uses .pkgs https://github.com/NixOS/nixpkgs/pull/354836/files#diff-29cf5f290eff451b0a09749b0bacb0d55694bcb1912561aeeb9e9ea178e05636R57

Reply:
Yes, this allows it to be put into the python3Packages set itself, which is needed to build several packages. If I didn't change this, the python version couldn't be overridden when it's added to python3Packages.